### ISSUE TITLE
Pyright crashing when not setting a type argument for `TypeVarTuple` when `TypeVar` argument is provided

### ISSUE THREAD

**Issue by kennipj** [microsoft/pyright#2583]
With following code sample:

```py
from typing import Generic, TypeVar
from typing_extensions import TypeVarTuple, Unpack

T = TypeVar("T")
Ts = TypeVarTuple("Ts")

class SomeClass(Generic[T, Unpack[Ts]]):
    ...


SomeClass[T]
```

pyright crashes with:
```
An internal error occurred while type checking file "file_name.py": TypeError: Cannot read properties of undefined (reading 'node')
    at adjustTypeArgumentsForVariadicTypeVar (/usr/local/lib/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:4965:57)
    at callback (/usr/local/lib/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:5165:36)
    at callback (/usr/local/lib/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:2729:39)
    at doForEachSubtype (/usr/local/lib/node_modules/pyright/dist/pyright-internal/src/analyzer/typeUtils.ts:305:9)
    at expandSubtype (/usr/local/lib/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:2722:13)
    at mapSubtypesExpandTypeVars (/usr/local/lib/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:2753:13)
    at getTypeFromIndexWithBaseType (/usr/local/lib/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:5071:22)
    at getTypeFromIndex (/usr/local/lib/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:4877:33)
    at getTypeOfExpression (/usr/local/lib/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:754:30)
    at hn (/usr/local/lib/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:14891:13)
Error performing analysis: TypeError: Cannot read properties of undefined (reading 'node')
    at adjustTypeArgumentsForVariadicTypeVar (/usr/local/lib/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:4965:57)
    at callback (/usr/local/lib/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:5165:36)
    at callback (/usr/local/lib/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:2729:39)
    at doForEachSubtype (/usr/local/lib/node_modules/pyright/dist/pyright-internal/src/analyzer/typeUtils.ts:305:9)
    at expandSubtype (/usr/local/lib/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:2722:13)
    at mapSubtypesExpandTypeVars (/usr/local/lib/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:2753:13)
    at getTypeFromIndexWithBaseType (/usr/local/lib/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:5071:22)
    at getTypeFromIndex (/usr/local/lib/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:4877:33)
    at getTypeOfExpression (/usr/local/lib/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:754:30)
    at hn (/usr/local/lib/node_modules/pyright/dist/pyright-internal/src/analyzer/typeEvaluator.ts:14891:13)
```

Python version is set to `3.10` in pyright config. Tested on pyright `1.1.188`

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.189, which I just published. It will also be included in the next release of pylance.
